src/features/validation/singleFieldValidationSagas.test.ts

```diff
@@ -27,6 +29,7 @@ describe('singleFieldValidationSagas', () => {
 
   beforeEach(() => {
     mockState = getInitialStateMock();
+    window.reduxStore.getState = jest.fn(() => mockState);
```

Comment:
Again, I strongly believe we should not start relying on this function. Using it directly deep inside functions removes the possibility to make these functions reactive.